nucypher/utilities/gas_strategies.py

```diff
@@ -57,7 +57,7 @@ def datafeed_fallback_gas_price_strategy(web3: Web3, transaction_params: TxParam
 #
 
 
-__RAW_WEB3_GAS_STRATEGIES = {'glacial': time_based.glacial_gas_price_strategy,  # 24h
+__RAW_WEB3_GAS_STRATEGIES = {# 'glacial': time_based.glacial_gas_price_strategy,  # 24h  # Glacial strategy is just a risk
```

Comment:
Haha "glacier of doom".

Reply:
"_If this glacier goes slower than one mile a year, we're all dead!_"

Reply:
:heavy_check_mark: